core/src/components/item/test/item.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import { h } from '@stencil/core';
+import { newSpecPage } from '@stencil/core/testing';
+
+import { Item } from '../item';
+
+it('should change focusable option after switching button option status', async () => {
```

Comment:
I will refactor this when I address the parent comment, for tests we try to nest them under a describe block for the component name. 

For example:
```ts
describe('item', () => {
  /* existing test */
});
```

It helps when reviewing the logs locally or in CI to know exactly which component the spec test is responsible for.

Reply:
Ah, got it! 👍